servicetest/networkgateway/test_network_gateway.py

```diff
@@ -110,13 +110,18 @@ def test_network_policy_drop(self):
         """
         try:
             sc = Container()
-            sc.start(DATA)
-            sc.set_gateway_config("network", GW_CONFIG_POLICY_DROP)
-            sc.launch_command("python " +
+            success = sc.start(DATA)
+            assert success is True
+            
+            success = sc.set_gateway_config("network", GW_CONFIG_POLICY_DROP)
+            assert success is True
+
+            pid, success = sc.launch_command("python " +
                               sc.get_bind_dir() +
                               "/testhelper.py" +
                               " --test-dir " + sc.get_bind_dir() +
                               " --do-ping example.com")
+            assert success is True
```

Comment:
yes but following snippet seems even worse 
`launched = sc.launch_command("true")
 assert launched == -1`

So lets keep success return but don't assert them when its not mandatory